### ISSUE TITLE
How can we help with this?

### ISSUE THREAD

**Issue by RealAmethyst** [parzival-space/rodecaster-utility#6]
A bit of an odd title, but I am really excited for this. I am a completely blind computer user and use a screen reader. On Windows the official app does not work well with it, so seeing that there is a Linux one in the works is exciting. I hope it can be made screen reader accessible when the day comes that this releases. I do use a rødecaster pro II and I would love to help out if possible. I am staying on Windows now though
